### PR TITLE
chore: Config updates to enable easier standalone_provider usage

### PR DESCRIPTION
**Pull request by daveconde**
# Config updates to enable easier standalone_provider usage

- Adds ability to pass SUPPORTED_CLOUD_PROVIDERS to make deploy/service
- Loads KUBE_CONFIG and image-pull.dockerconfigjson secrets as base64 to make it easier to pass as multi line ENV_VARS

### CODE REVIEW

**Review by machi1990 — CHANGES_REQUESTED** [review]
Thank you so much @daveconde for this improvement. I've left two comments.

I do not think we need to worry about making sure the change is backward compatible. What do you reckon @JameelB ?

File: Makefile
Comment (machi1990) [inline Makefile:802]: Can we add a line or two on the https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/539b3ee14b821c3b642eac549eda47de01a97d4b/docs/deploying-kas-fleet-manager-to-openshift.md#deploy-kas-fleet-manager docs about this variable and what the default is?

File: templates/secrets-template.yml
Comment (daveconde) [reply templates/secrets-template.yml:136]: Thanks @machi1990, I've updated the docs

File: Makefile
Comment (daveconde) [reply Makefile:802]: 👍

File: templates/secrets-template.yml
Comment (daveconde) [reply templates/secrets-template.yml:136]: I've added an auto base64 encoding if the contents is read from the file instead of the env var.

Comment (machi1990) [reply templates/secrets-template.yml:136]: Thanks

Comment (machi1990) [reply templates/secrets-template.yml:136]: looks good to me, thanks!

**Review by machi1990 — APPROVED** [review]
lgtm

let's get an additional review from another team member before merging.
/cc @JameelB would you have some time to have a look at this? Thanks

**Review by JameelB — APPROVED** [review]
Looks good to me too! Thanks for this @daveconde!

**Pull request merged**
